pkg/spdk/engine.go

```diff
@@ -215,11 +215,16 @@ func (e *Engine) Create(spdkClient *spdkclient.Client, replicaAddressMap map[str
 		e.IP = targetIP
 
 		// Get ReplicaModeMap and ReplicaBdevNameMap
-		targetSPDKClient, err := GetServiceClient(net.JoinHostPort(e.IP, strconv.Itoa(types.SPDKServicePort)))
+		targetSPDKServiceAddress := net.JoinHostPort(e.IP, strconv.Itoa(types.SPDKServicePort))
+		targetSPDKClient, err := GetServiceClient(targetSPDKServiceAddress)
 		if err != nil {
 			return nil, err
 		}
-		defer targetSPDKClient.Close()
+		defer func() {
+			if errClose := targetSPDKClient.Close(); errClose != nil {
+				e.log.WithError(errClose).Errorf("Failed to close target spdk client with address %s", targetSPDKServiceAddress)
```

Comment:
NIT: `"Failed to .... during engine create"`
Similar below.

This helps quickly pinpoint which function has trouble with closing replica clients

Reply:
Updated. Thank you.